src/battle_ai_main.c

```diff
@@ -1034,7 +1034,7 @@ static s32 AI_CheckBadMove(u32 battlerAtk, u32 battlerDef, u32 move, s32 score)
 // the following checks apply to any target (including user)
 
     // throat chop check
-    if (gDisableStructs[battlerAtk].throatChopTimer && IsSoundMove(move))
+    if (gDisableStructs[battlerAtk].throatChopTimer > gBattleTurnCounter && IsSoundMove(move))
```

Comment:
to me `gDisableStructs[battlerAtk].throatChopTimer > gBattleTurnCounter` feels less intuitive than the timer ticking to 0. does this fix a bug?